documentation/feature/medical-system.md

```diff
@@ -6,238 +6,370 @@ group: feature
+- Sources: vehicle crashes, grenades, explosions, artillery shells, backblast, stabs
+- Effects: pain - light, bleeding - speed depends on length and size of the wound.
+
+<p align="center">
+<b>Lacerations (tears)</b>
+</p>  
+
+- these are separating wounds that produce ragged edges. They are produced by a tremendous force against the body, either from an internal source as in childbirth, or from an external source like a punch.
+- Sources: vehicle crashes, punches
+- Effects: pain - light, bleeding - slow to medium speed (depends on wound size).
+
+<p align="center">
+<b>Velocity wound</b>
+</p>  
+
+- They are caused by an object entering the body at a high speed, typically a bullet or small pieces of shrapnel.
+- Sources: bullets, grenades, explosions, artillery shells.
+- Effects: pain - extremely high, bleeding - medium speed (depends on wound size).
+
+<p align="center">
+<b>Puncture wounds</b>
+</p>  
+
+- Deep, narrow wounds produced by sharp objects such as nails, knives, and broken glass.
+- Sources: stabs, grenades.
+- Effects: pain - light, bleeding - slowly.
+
+In order to stop the bleeding, all injuries on every body part requires treatment. This is done by either applying a tourniquet to legs or arms as a temporary solution, or by using bandages to stop the bleeding as a more permanent fix. 
+<p align="center">
+<b>Basic </b>
+</p>
+
+Wound type | Effectiveness
+---------- | ---------- |
+Abrasions | highest
+Avulsions | lowest
+Contusions | highest
+Crush wounds | low
+Cut wounds | very low
+Lacerations | medium
+Velocity wounds | lowest
+Puncture wounds| low
+
+
+<p align="center">
+<b>Bandage (packing)</b>
+</p>
+
+Wound type | Effectiveness
+---------- | ---------- |
+Abrasions | highest
+Avulsions | highest
+Contusions | highest
+Crush wounds | low
+Cut wounds | lowest
+Lacerations | lowest
+Velocity wounds | highest
+Puncture wounds | lowest
+
+<p align="center">
+<b>Bandage (elastic)</b>
+</p>
+
+Wound type | Effectiveness
+---------- | ---------- |
+Abrasions | highest
+Avulsions | lowest
+Contusions | highest
+Crush wounds | highest
+Cut wounds | highest
+Lacerations | highest
+Velocity wounds | low
+Puncture wounds | high
+
+<p align="center">
+<b>QuikClot</b>
+</p>
+
+Wound type | Effectiveness
+---------- | ---------- |
+Abrasions | medium
+Avulsions | high
+Contusions | medium
+Crush wounds | medium
+Cut wounds | medium
+Lacerations | medium
+Velocity wounds | high
+Puncture wounds | high
+
+<p align="center">
+<b>Tourniquet</b>
+</p>
+
+- Can only be applied on limbs
+- Stops bleeding from wounds
+- Should be taken off as fast as possible and applied only to give medic time to bandage all the wounds
+- If not taken off for a while it will cause pain to patient, can cause death that way
+
+<p align="center">
+<b>IVs</b>
+</p>
+
+IV | Effect
+---------- | ---------- |
+Saline and blood | restore the blood of the patient
+Plasma | raise the blood pressure of the patient
```

Comment:
At the moment all 3 options restore the volume of liquid in the blood stream. As a result, blood pressure is raised for all of them, provided it goes in faster as it goes out (not bleeding). Raising BP is a side effect of any IV we have.